index.html

```diff
@@ -22,17 +22,329 @@
+        <nav class="container11">
+            <ul class="data3">
+                <li class="Lang2"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container14">
+            <ul class="navlink2">
+                <li class="project1"><a href="#">See this Project</a></li>
+                <li class="arrow"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container15">
+        <ul class="comp2">
+            <li class="rect2"><img src="imgformobile/Rectangle2.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container16">
+        <ul>
+            <li class="Proj2"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container17">
+            <ul class="data4">
+                <li class="Lang3"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container18">
+            <ul class="data5">
+                <li class="Lang4"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container19">
+            <ul class="data6">
+                <li class="Lang5"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container20">
+            <ul class="navlink3">
+                <li class="project2"><a href="#">See this Project</a></li>
+                <li class="arrow1"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container21">
+        <ul class="comp3">
+            <li class="rect3"><img src="imgformobile/Rectangle3.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container22">
+        <ul>
+            <li class="Proj3"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container23">
+            <ul class="data7">
+                <li class="Lang6"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container24">
+            <ul class="data8">
+                <li class="Lang7"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container25">
+            <ul class="data9">
+                <li class="Lang8"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container26">
+            <ul class="navlink4">
+                <li class="project3"><a href="#">See this Project</a></li>
+                <li class="arrow2"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container27">
+        <ul class="comp4">
+            <li class="rect4"><img src="imgformobile/Rectangle4.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container28">
+        <ul>
+            <li class="Proj4"><a href="#">Hello-World</a></li>
```

Comment:
- [ ] Kindly change this to a paragraph tag because this will give ore information about the card  👍🏿